### PR TITLE
remove freezer references (bsc#1122905,SCPM-102)

### PR DESCRIPTION
**Pull request by csymons-suse**
planning guide

### CODE REVIEW

File: xml/planning-core_non-core_openstack_support.xml [hunk 1]
```diff
@@ -47,81 +47,81 @@
...
       </row>
       <row>
-       <entry>Octavia</entry>
-       <entry>Yes</entry>
-       <entry>Yes</entry>
-       <entry>  </entry>
        <entry>&swift;</entry>
        <entry>Yes</entry>
        <entry>Yes</entry>
+       <entry>  </entry>
```
Comment (dmpop) [inline xml/planning-core_non-core_openstack_support.xml]: Out of curiosity: What are these  empty `<entry></entry>`?

Comment (asettle) [reply xml/planning-core_non-core_openstack_support.xml]: I imagine they generate the remainder of the table? Correct @csymons-suse ?

Comment (asettle) [reply xml/planning-core_non-core_openstack_support.xml]: Oh potentially not, it doesn't generate properly

Comment (csymons-suse) [reply xml/planning-core_non-core_openstack_support.xml]: It builds properly with DAPS. In the rendered table, it inserts a small blank column that makes the table readable. This was a conversion from an ugly artifact in the original HOS documentation. I tried it with and without the blank column. Preferred this look.

Might be a moot point as there are now "conflicts that must be resolved". This pull request is almost a month old. Ungood.

Comment (dmpop) [reply xml/planning-core_non-core_openstack_support.xml]: I see. Thanks for the explanation!

**Comment by csymons-suse** [comment]
superseded by
https://github.com/SUSE-Cloud/doc-cloud/pull/782
https://github.com/SUSE-Cloud/doc-cloud/pull/783
https://github.com/SUSE-Cloud/doc-cloud/pull/784

**Pull request closed without merging**